.readthedocs.yaml

```diff
@@ -6,7 +6,7 @@ version: 2
 
 # Set the OS, Python version and other tools you might need
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
```

Comment:
This is unrelated, please, revert, I usually like to stick to the recommended version in the docs: https://docs.readthedocs.io/en/stable/config-file/v2.html